tests/framework/unit_tests/utils/testRandomUtils.py

```diff
@@ -112,34 +112,68 @@ def checkType(comment,value,expected,updateResults=True):
 # Reseed at the beginning of sections and add new tests to the end of sections.
 
 # set the stochastic environment TODO check both someday?
+# cannot pass the numpy as the stochasticEnv
 randomUtils.stochasticEnv = 'crow'
 
+eng = randomUtils.newRNG()
 # randomSeed(), setting the random seed
-randomUtils.randomSeed(42)
+randomUtils.randomSeed(42,engine=None)
+randomUtils.randomSeed(42,engine=eng)
 # check that seed is set
-checkAnswer('First float from first seed',randomUtils.random(),0.374540118847)
+checkAnswer('First float from first seed',randomUtils.random(engine=None),0.374540118847)
+checkAnswer('First float from first seed',randomUtils.random(engine=eng),0.374540118847)
```

Comment:
It'd be nice if the tests with the provided engine had a slightly different check message, so we can find the erroring check if one does fail; maybe just by adding `, local engine` or `, eng provided` to the string?

Reply:
Added engine check message in each check answer.